src/cheri/app-cap-description.adoc

```diff
@@ -175,30 +175,27 @@ orthogonal to permissions as it can vary arbitrarily using <<SCMODE>>.
 | 4-5   | ✔ | ✔ | ✔ | ✔  | ✔  | 1   | ✔ |     | Mode^1^  | Execute + Data & Cap RW
 | 6-7   | ✔ | ✔ |   |    |    | 0^2^| ✔ |     | Mode^1^  | Execute + Data RW
 11+| *Quadrant 2: Restricted capability data read/write*
-11+| bit[2] = write, bit[1:0] = store level. R and C implicitly granted, LM dependent on W permission.
+11+| bit[2] = write, bit[1] reserved, bit[0] = SL. R and C implicitly granted, LM dependent on W permission.
 |Encoding[2:0]| R | W | C | LM | LG | SL    | X | ASR | Mode^1^ |
 | 0-2   10+| reserved
-| 3       | ✔ |   | ✔ |    |    | 0^1^  |   |     | N/A | Data & Cap R0 (without <<lm_perm>>)
```

Comment:
Looks like the change from bits[]->encoding has made this table misaligned and a bit harder to read. I can fix this but don't want to conflict.